src/components/models/Block3HowTo.js

```diff
@@ -24,21 +27,22 @@ const useStyles = makeStyles({
   },
 });
 
-const Block3HowTo = ({ title, subtitle, markImage, processItems }) => {
-  const { muiGridBlockContainer, itemContainer, textContainer, body2, buttonLink, icon, markImg } = useStyles();
+const Block3HowTo = ({ title: title2, subtitle, markImage, processItems }) => {
```

Comment:
Rather than just making this `title2`, let's name it something more descriptive - e.g. `blockTitle`.